sdk/tables/azure-tables/review/tables.api.review.md

```diff
@@ -0,0 +1,695 @@
+  version?: string;
+}
+
+// @public
+export interface TableDeleteHeaders {
+  clientRequestId?: string;
+  date?: Date;
+  requestId?: string;
+  version?: string;
+}
+
+// @public
+export interface TableEntityContinuationToken {
+  nextPartitionKey?: string;
+  nextRowKey?: string;
+}
+
+// @public
+export interface TableEntityQueryResponse {
+  odataMetadata?: string;
+  value?: {
+    [propertyName: string]: any;
+  }[];
+}
+
+// @public
+export interface TableGetAccessPolicyHeaders {
+  clientRequestId?: string;
+  date?: Date;
+  requestId?: string;
+  version?: string;
+}
+
+// @public
+export interface TableInsertEntityHeaders {
+  clientRequestId?: string;
+  contentType?: string;
+  date?: Date;
+  eTag?: string;
+  preferenceApplied?: string;
+  requestId?: string;
+  version?: string;
+}
+
+// @public
+export interface TableInsertEntityHeaders {
+  clientRequestId?: string;
+  contentType?: string;
+  date?: Date;
+  eTag?: string;
+  preferenceApplied?: string;
+  requestId?: string;
+  version?: string;
+}
+
+// @public
+export interface TableInsertEntityOptionalParams extends coreHttp.OperationOptions {
+  queryOptions?: GeneratedQueryOptions;
+  requestId?: string;
+  responsePreference?: ResponseFormat;
+  tableEntityProperties?: {
+    [propertyName: string]: any;
+  };
+  timeout?: number;
+}
+
+// @public
+export interface TableMergeEntityHeaders {
+  clientRequestId?: string;
+  date?: Date;
+  eTag?: string;
+  requestId?: string;
+  version?: string;
+}
+
+// @public
+export interface TableMergeEntityOptionalParams extends coreHttp.OperationOptions {
+  ifMatch?: string;
+  queryOptions?: GeneratedQueryOptions;
+  requestId?: string;
+  tableEntityProperties?: {
+    [propertyName: string]: any;
+  };
+  timeout?: number;
+}
+
+// @public
+export interface TableQueryEntitiesHeaders {
+  clientRequestId?: string;
+  date?: Date;
+  requestId?: string;
+  version?: string;
+  xMsContinuationNextPartitionKey?: string;
+  xMsContinuationNextRowKey?: string;
+}
+
+// @public
+export interface TableQueryEntitiesOptionalParams extends coreHttp.OperationOptions {
+  nextPartitionKey?: string;
+  nextRowKey?: string;
+  queryOptions?: GeneratedQueryOptions;
+  requestId?: string;
+  timeout?: number;
+}
+
+// @public
+export type TableQueryEntitiesResponse = TableQueryEntitiesHeaders &
+  TableEntityQueryResponse & {
+    _response: coreHttp.HttpResponse & {
+      bodyAsText: string;
+      parsedBody: TableEntityQueryResponse;
+      parsedHeaders: TableQueryEntitiesHeaders;
+    };
+  };
+
+// @public
+export interface TableQueryEntitiesWithPartitionAndRowKeyHeaders {
+  clientRequestId?: string;
+  date?: Date;
+  eTag?: string;
+  requestId?: string;
+  version?: string;
+  xMsContinuationNextPartitionKey?: string;
+  xMsContinuationNextRowKey?: string;
+}
+
+// @public
+export type TableQueryEntitiesWithPartitionAndRowKeyResponse = TableQueryEntitiesWithPartitionAndRowKeyHeaders & {
+  [propertyName: string]: any;
```

Comment:
Note: `[propertyName: string]: any` has been overridden above in `GetEntityResponse` to be a Generic type